boot/bootutil/include/bootutil/image.h

```diff
@@ -74,6 +74,7 @@ struct flash_area;
  *   2nd one is the actual signature.
  */
 #define IMAGE_TLV_KEYHASH           0x01   /* hash of the public key */
+#define IMAGE_TLV_KEY               0x02   /* public key */
```

Comment:
I wonder if we should call this PUBKEY. I realize that KEYHASH already doesn't have that, and we aren't going to be storing the private key. It just feels weird to call it just "KEY" though.

Reply:
I understand what you mean, I've renamed it to `PUBKEY`.